protobufs/livekit_agent.proto

```diff
@@ -21,154 +21,174 @@ option ruby_package = "LiveKit::Proto";
 
 import "livekit_models.proto";
 
-message WorkerInfo {
-    string id = 1;
-    string namespace = 2;
-    string version = 3;
-    string name = 4;
-    JobType type = 5;
-    ParticipantPermission allowed_permissions = 6;
+service AgentService {
+  rpc StartAgentJob(StartAgentJobRequest) returns (Job);
+  rpc StopAgentJob(StopAgentJobRequest) returns (Job);
```

Comment:
when we stop a job how are we going to signal this to the agent? cut their participant session? i want to be sure i'm handling this the same in cloud as oss

Reply:
In a first implementation where there is no adversarial relationship between the room and the agent (as they belong to the same entity), I think adding a JobTermination ServerMessage may be enough. For better security and resilience, we should probably eventually enforce this by terminating the room/participant RTC connection to the agent indeed.

Reply:
that works too - we could set a timeout and kill the peer connection if it doesn't close on its own but giving the client an opportunity to close gracefully (ie after a keyframe) is a good idea.

Reply:
We have a `RemoveParticipant` API that would end up sending a `LEAVE` signal event to the user.. is that sufficient for this?

Reply:
I think it's a bit of a roundabout way of doing it. I typically rather have explicit APIs. Particularly, if there is a `Start` API, I think a `Stop` is typically expected.

That being said, we should define the behavior if RemoveParticipant is called on the agent participant.

Reply:
Totally agreed with an explicit API.. I'm more talking about "how to actually stop the agent".. you could use the same mechanisms as `RemoveParticipant`